From 5e4fe0eb54c3ae2cf71f25732ec7d60e7cbb0b5d Mon Sep 17 00:00:00 2001
From: Yuta Saito <kateinoigakukun@gmail.com>
Date: Tue, 25 Jul 2023 05:46:39 +0000
Subject: [PATCH] [wasm][test] Skip crash tests on WASI for now

WASI doesn't support spawning a subprocess, so crash tests crashes the
test harness itself. Skip them for now.
---
 test/stdlib/UnsafePointer.swift.gyb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/stdlib/UnsafePointer.swift.gyb b/test/stdlib/UnsafePointer.swift.gyb
index cc1775d78a5..08b84a61c4e 100644
--- a/test/stdlib/UnsafePointer.swift.gyb
+++ b/test/stdlib/UnsafePointer.swift.gyb
@@ -537,6 +537,7 @@ ${SelfName}TestSuite.test("pointer(to:)") {
 % end
 }
 
+#if !os(WASI)
 ${SelfName}TestSuite.test("pointer(to:).overflow") {
   struct Example {
     var a = false
@@ -558,6 +559,7 @@ ${SelfName}TestSuite.test("pointer(to:).overflow") {
   let doublePointer = p.pointer(to: \.d)
   expectNotNil(doublePointer)
 }
+#endif
 
 % end
 
-- 
2.40.0

